mitype/keycheck.py

```diff
@@ -16,7 +16,8 @@ def is_escape(key):
             KEY_UP or ^G.
 
     Returns:
-        bool: Returns `True` if pressed key is ESC key.
+        bool:
+            Returns `True` if pressed key is ESC key.
             Returns `False` otherwise.
```

Comment:
Even though this method works as expected, for sake of consistency across other docstrings, we start the description on the same line as the return type.
```
    Returns:
        bool: Returns `True` if pressed key is ESC key.
        Returns `False` otherwise.
```

Reply:
Alright I'll make the required changes